components/Cards/SchoolCard/__tests__/BackSchoolCard.test.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import createShallowSnapshotTest from 'test-utils/createShallowSnapshotTest';
+import BackSchoolCard from '../BackSchoolCard';
+
+const locations = [
+  {
+    va_accepted: true,
+    address1: '825 Battery Street',
+    address2: '3rd Floor',
+    city: 'San Francisco',
+    zip: '94111',
+  },
+];
+
+describe('BackSchoolCard', () => {
+  it('should render with required props', () => {
+    createShallowSnapshotTest(
+      <BackSchoolCard cardFlipCallback={jest.fn()} locations={locations} logoSource="logoSource">
+        <p>Testing!</p>
```

Comment:
so nested elements within a component would be considered the same thing as the `children` prop.

While we provide it as a default for the `createComponent` script, it's unnecessary here (or in any of your school card-related tests since none of those components deal with the `children` prop.)

See below for an example